www-client/uget/uget-2.2.3.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit gnome2-utils xdg-utils
+
+DESCRIPTION="Download manager using gtk+ and libcurl"
+HOMEPAGE="http://www.ugetdm.com"
+SRC_URI="mirror://sourceforge/urlget/${P}-1.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ppc ~x86"
+IUSE="aria2 appindicator control-socket +gnutls gstreamer libnotify nls openssl rss"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-fno-common-713812.diff"
+)
+
+REQUIRED_USE="^^ ( gnutls openssl )"
```

Comment:
We have updated the logic to control ssl usage, you can grasp the general idea here:
https://wiki.gentoo.org/wiki/Project:LibreSSL

Reply:
Note that you don't need this `REQUIRED_USE` and that we try to avoid them as much as possible. You can find a logic to use from the page I linked, 
https://wiki.gentoo.org/wiki/Project:LibreSSL

Basically you have `gnutls libressl ssl` USES and it's prioritized as `gnutls > libressl > openssl`.